Helper/Order.php

```diff
@@ -802,17 +803,18 @@ public function applyExternalQuoteData($quote) {
 
     /**
      * @param OrderInterface $order
-     * @param Quote $quote
+     * @param Quote $immutableQuote
      */
-    public function dispatchPostCheckoutEvents($order, $quote) {
+    public function dispatchPostCheckoutEvents($order, $immutableQuote) {
```

Comment:
This method is called with the parent / session quote, not the immutable / cloned one.